pkg/ccl/backupccl/show.go

```diff
@@ -324,9 +325,26 @@ func showBackupPlanHook(
 		}
 		defer store.Close()
 
+		// A user that calls SHOW BACKUP TO <incremental_dir> on an encrypted incremental
```

Comment:
nit: drop the TO

Reply:
ooof. apologies for my sloppiness. thanks for catching these little nits!